ember-cli-build.js

```diff
@@ -6,6 +6,12 @@ const EmberApp = require("ember-cli/lib/broccoli/ember-app");
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
+    fontawesome: {
+      icons: {
+        "free-regular-svg-icons": "all",
+        "free-solid-svg-icons": "all",
```

Comment:
Its probably worth it to actually just specify the ones you need since the whole fontawesome library is huuuge.

Reply:
**With all the icons:**
File sizes:
...
 - dist/assets/vendor-eefece088a2eb0380cdeba910ecbb960.js: 2.79 MB (771.51 KB gzipped)
---
**With only one icon**
File sizes:
...
 - dist/assets/vendor-591e0f84c26680c8f564049bcccc5512.js: 1.76 MB (448.44 KB gzipped)
---
**yup, 1.3 MB difference :+1:**

Reply:
Final size with all the needed icons:
 - dist/assets/vendor-27f809b2ab2c2fd7b6204c58f702c19e.js: 1.78 MB (455.03 KB gzipped)

Reply:
`done`